docs/source/contributing/docker_container.md

```diff
@@ -0,0 +1,21 @@
+(pr_checklist)=
+# Running PyMC in Docker
+
+We have provided a Dockerfile which helps for isolating build problems, and local development.
+Install [Docker](https://www.docker.com/) for your operating system, clone this repo, then
+run `./scripts/docker_container.sh build` to build a `pymc` docker image.
+
+After successfully building the docker image, you can start a local docker container called `pymc` from `bash` or from [`jupyter`](http://jupyter.org/) notebook server running on port 8888.
+
+```bash
+$  bash scripts/docker_container.sh bash # running the container with bash
+$  bash scripts/docker_container.sh bash # running the container on jupyter notebook
+```
+
+The repo will be running the code from your local copy of `pymc`, so it is good for development. You may also use it to run the test suite, with
+
+```bash
+$  bash scripts/docker_container.sh bash # logon to the container
+$  cd ./pymc/tests
+$  . ./../../scripts/test.sh # takes a while!
```

Comment:
In addition to the dollar sign comment, do we use these `scripts/test.sh` in CI? Shouldn't the advise be to use pytest commands explicitly like we explain on https://www.pymc.io/projects/docs/en/stable/contributing/running_the_test_suite.html? We also recommend not running the whole test suite all at once unless necessary which that would seem to point against.

Reply:
Yes, this content is from the old docs. I will remove them then. Also, not sure if the `scripts/test.sh` is used in any CI. If it is not used, we should also remove this script.